src/components/connection/AddDatabaseForm.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="save-connection expand">
+    <h3 class="dialog-c-title">Add database</h3>
+
+    <div v-if="error" class="alert alert-danger">
+      {{error}}
+    </div>
+
+    <div class="form-group">
+      <input class="form-control" v-model="databaseName" @keydown.enter.prevent.stop="save" type="text" placeholder="Database Name">
+    </div>
+
+    <div class="form-group">
+      <select v-model="selectedCharset">
+        <option v-for="charset in charSets" v-bind:key="charset" :selected="charset === defaultCharSet" :value="charset">{{charset}}</option>
+      </select>
+    </div>
+
+    <div class="form-group">
+      <select v-model="selectedCollation">
+        <option v-for="collation in collations" v-bind:key="collation" :value="collation">{{collation}}</option>
+      </select>
+    </div>
+
+    <div class="save-actions">
+      <button class="btn btn-flat" @click.prevent="$emit('cancel')">Cancel</button>
+      <button class="btn btn-primary save" @click.prevent="save">Add</button>
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  props: ['connection'],
+  data() {
+    return {
+      charSets: [],
+      collations: [],
+      defaultCharSet: 'utf8mb4',
```

Comment:
Is this default supported across all database types?

We should get this value from the connection too, like `await connection.getDefaultCharset`
Probably we also need a `connection.supportsDatabaseCreation` or something

For example: Sqlite3 cannot create new databases, becuase a file === database.

Reply:
This is currently hardcoded (and should be get from the database) as i first wanted some feedback for my other changes (as this is my first vue/electron project)

Reply:
Yep! I figured.

Your Vue code is great! Thanks for working so hard to make the changes good.

Reply:
done